cli/flox/doc/flox-activate.md

```diff
@@ -90,11 +90,14 @@ See [`manifest.toml(5)`](./manifest.toml.md) for more details on shell hooks.
    otherwise a warning will displayed and activation will continue.
 
    This flag is currently incompatible with "in-place" activations and remote
-   environnments, but these features will be added in the future.
+   environments, but these features will be added in the future.
 
    The services started with this flag will be cleaned up once the last
    activation of this environment terminates.
 
+   A remote environment can only have a single instance of services, regardless
+   of where or how many times the environment is invoked concurrently.
```

Comment:
```suggestion
   of where or how many times the environment is activated concurrently.
```

Reply:
Done, along with the changes in this thread: https://flox-dev.slack.com/archives/C06T9RL3BQT/p1725466291320849

> I'm going to:
> - replace "instance" with "set of" to indicate that it's the collection of services for an environment and differentiate if we use `instances` as a manifest option later
> - include "running" to indicate that it's about starting them rather than the definition of them
> - remove "where" because it felt awkward to combine and is just a subset of concurrently activating a remote environment
> 
> Here's the reworded paragraph:
> > A remote environment can only have a single set of running services, regardless of how many times the environment is activated concurrently.
> 
> There didn't seem to be any bites on adding tutorial content so I'll leave that for now.
> 
> I forgot to mention previously that I'm also not going to hint in the docs that we intend to solve this with composition later because we don't know when that will be prioritised yet.